client/src/send_and_confirm_transactions_in_parallel.rs

```diff
@@ -54,21 +55,74 @@ struct BlockHashData {
     pub last_valid_block_height: u64,
 }
 
+// Deprecated struct to maintain backward compatibility
+#[deprecated(note = "Use SendAndConfirmConfigV2 with send_and_confirm_transactions_in_parallel_v2")]
 #[derive(Clone, Debug, Copy)]
 pub struct SendAndConfirmConfig {
     pub with_spinner: bool,
     pub resign_txs_count: Option<usize>,
 }
 
-/// Sends and confirms transactions concurrently in a sync context
+// New struct with RpcSendTransactionConfig for non-breaking change
+#[derive(Clone, Debug, Copy)]
+pub struct SendAndConfirmConfigV2 {
+    pub with_spinner: bool,
+    pub resign_txs_count: Option<usize>,
+    pub rpc_send_transaction_config: RpcSendTransactionConfig,
+}
+
+#[deprecated(note = "Use send_and_confirm_transactions_in_parallel_v2")]
+pub async fn send_and_confirm_transactions_in_parallel<T: Signers + ?Sized>(
+    rpc_client: Arc<RpcClient>,
+    tpu_client: Option<QuicTpuClient>,
+    messages: &[Message],
+    signers: &T,
+    config: SendAndConfirmConfig,
+) -> Result<Vec<Option<TransactionError>>> {
+    let config_v2 = SendAndConfirmConfigV2 {
+        with_spinner: config.with_spinner,
+        resign_txs_count: config.resign_txs_count,
+        rpc_send_transaction_config: RpcSendTransactionConfig {
+            skip_preflight: false,
+            ..RpcSendTransactionConfig::default()
+        },
+    };
+    send_and_confirm_transactions_in_parallel_v2(
+        rpc_client, tpu_client, messages, signers, config_v2,
+    )
+    .await
+}
+
+#[deprecated(note = "Use send_and_confirm_transactions_in_parallel_blocking_v2")]
 pub fn send_and_confirm_transactions_in_parallel_blocking<T: Signers + ?Sized>(
     rpc_client: Arc<BlockingRpcClient>,
     tpu_client: Option<QuicTpuClient>,
     messages: &[Message],
     signers: &T,
     config: SendAndConfirmConfig,
 ) -> Result<Vec<Option<TransactionError>>> {
-    let fut = send_and_confirm_transactions_in_parallel(
+    let config_v2 = SendAndConfirmConfigV2 {
+        with_spinner: config.with_spinner,
+        resign_txs_count: config.resign_txs_count,
+        rpc_send_transaction_config: RpcSendTransactionConfig {
+            skip_preflight: false,
+            ..RpcSendTransactionConfig::default()
+        },
```

Comment:
nit: can just go straight with default here
```suggestion
        rpc_send_transaction_config: RpcSendTransactionConfig::default(),
```